temporal-normalizer/src/test/java/edu/illinois/cs/cogcomp/temporal/normalizer/tests/TestTemporalChunker.java

```diff
@@ -0,0 +1,365 @@
+            TextAnnotation ta = pipeline.createAnnotatedTextAnnotation("corpus", "id", testText.get(j));
+//            TextAnnotation ta = null;
+//            try {
+//                ta = annotator.createBasicTextAnnotation("corpus", "id", testText.get(j));
+//            } catch (AnnotatorException e) {
+//                fail("Exception while creating TextAnnotation" + e.getStackTrace());
+//            }
+//            try {
+//                annotator.addView(ta, ViewNames.POS);
+//            } catch (AnnotatorException e) {
+//                fail("Exception while adding POS VIEW " + e.getStackTrace());
+//            }
+            taList.add(ta);
+        }
+        System.out.println("Start");
+        long startTime = System.currentTimeMillis();
+        for (int j = 0; j < testText.size(); j ++) {
+//            if (!docIDs.get(j).contains("XIE19981203.0008")){
+//                continue;
+//            }
+            tca.addDocumentCreationTime(DCTs.get(j));
+            TextAnnotation ta = taList.get(j);
+            //System.out.println(docIDs.get(j));
+
+            try {
+                tca.addView(ta);
+            } catch (AnnotatorException e) {
+                fail("Exception while adding TIMEX3 VIEW " + e.getStackTrace());
+            }
+
+//            View timexView = ta.getView(ViewNames.TIMEX3);
+
+//            String corpId = "IllinoisTimeAnnotator";
+//            List<Constituent> timeCons = timexView.getConstituents();
+
+//            for(Constituent c: timeCons) {
+//                System.out.println(c);
+//            }
+            // Keep track of the compressed index of each constituent.
+/*            Span[] compressedSpans = new Span[timeCons.size()];
+            int spanStart;
+
+            // Builds a string of the concatenated constituents from a labeled view.
+            StringBuilder builder = new StringBuilder();
+            for (int i = 0; i < timeCons.size(); i++) {
+                Constituent c = timeCons.get(i);
+                spanStart = builder.length();
+
+                builder.append(c.toString());
+                builder.append("; ");
+
+                compressedSpans[i] = new Span(spanStart, builder.length() - 2);
+            }
+            String compressedText = builder.toString();
+            assertNotNull(compressedText);
+*/
+            String outputFileName = "./last_30_TB_chunker_illininorm/"+docIDs.get(j) + ".tml";
+//            for(TimexChunk tc: tca.getTimex()) {
+//                System.out.println(tc.toTIMEXString());
+//            }
+//            System.out.println("\n");
+            tca.write2Text(outputFileName, docIDs.get(j) ,testText.get(j));
+            tca.deleteTimex();
+        }
+        long endTime   = System.currentTimeMillis();
+        long totalTime = endTime - startTime;
+        System.out.println("Process time: " + totalTime);
+        System.out.println("Preprocess + process time: " + (endTime-preprocessTime) );
```

Comment:
There don't appear to be any assertions in these tests. The tests should verify that some specific annotations are present, and that their values are as you would expect.